### PR TITLE
Add uploadDate to Video schema.org

### LINKED ISSUE #1685: Add uploadDate to video schema.org
**Is your feature request related to a problem? Please describe.**
Video schema.org data should contain an uploadDate field.  This is currently not there.  This will be exposed in the Ares data as availableFrom.

**Describe the solution you'd like**
uploadDate field inserted into schema where the date is made available by Ares.

**Describe alternatives you've considered**
N/A

**Testing notes**
Inspecting Storybook to ensure schema metadata is added. 

Dev insight: Unit tests should be sufficient. Can view integration within Storybook's Video container. http://localhost:9001/?selectedKind=Video%20Container

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #1685 

**Overall change:** _Add uploadDate to video schema.org._

**Code changes:**

- _Retrieve uploadDate value from aresMetadata, and add it to video schema.org_
- _Update video fixture data by adding `availableFrom` to aresMetadata._
- _Update tests and snapshots._
- _Errors in the Structured Data tool depicted on #1678 are now gone_
structured data tool test

- _Locally, the updated ld+JSON script tag can be seen in the head of the Video container iframes in Storybook_
storybook

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/containers/Video/videoMetadata.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const videoMetadata = blocks => {
       description: deepGet(['model', 'synopses', 'short'], block),
       duration: deepGet(['model', 'versions', [0], 'duration'], block),
       thumbnailUrl: `https://${deepGet(['model', 'imageUrl'], block)}`,
+      uploadDate: deepGet(['model', 'versions', [0], 'availableUntil'], block),
```
Comment (sareh) [inline src/app/containers/Video/videoMetadata.js]: The field used in the JSON should be `availableFrom` - as was mentioned in the original issue. 

```suggestion
      uploadDate: deepGet(['model', 'versions', [0], 'availableFrom'], block),
```
An example video that has this field, for use in unit tests, can be this one: https://www.test.bbc.co.uk/news/articles/c4d8dqrd5wyo.json

**Review by twinlensreflex — DISMISSED** [review]
LGTM 👍

**Review by oluoluoxenfree — DISMISSED** [review]
👍

**Review by oluoluoxenfree — APPROVED** [review]

**Review by twinlensreflex — APPROVED** [review]
LGTM 👍

**Comment by jamesbrumpton** [comment]
Happy for this be merged.

**Pull request merged**